source/gui/settingsDialogs.py

```diff
@@ -1111,12 +1111,27 @@ def makeSettings(self, settingsSizer):
 		self.lineNumberCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report line &numbers"))
 		self.lineNumberCheckBox.SetValue(config.conf["documentFormatting"]["reportLineNumber"])
 		settingsSizer.Add(self.lineNumberCheckBox,border=10,flag=wx.BOTTOM)
-		# Translators: This message is presented in the document formatting settings dialogue
-		# If this option is selected, NVDA will cound the leading spaces and tabs of a line and speak it.
-		#
-		self.lineIndentationCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report l&ine indentation"))
-		self.lineIndentationCheckBox.SetValue(config.conf["documentFormatting"]["reportLineIndentation"])
-		settingsSizer.Add(self.lineIndentationCheckBox,border=10,flag=wx.BOTTOM)
+		sizer=wx.BoxSizer(wx.HORIZONTAL)
+		# Translators: This is the label for a combobox in the
+		# Document  Formatting  dialog (possible choices are Off, Speech, Tones, or Both.
+		sizer.Add(wx.StaticText(self,wx.ID_ANY,label=_("Report Line &Indents With:")))
+		indentChoices=[
+			#Translators: A setting to report No  line Indentation.
+			_("Off"),
+			#Translators: A setting to report indents with Speech.
+			_("Speech"),
+			#Translators: A setting to report indents with tones.
+			_("Tones"),
+			#Translators: A setting to report indents with both tones and Speech.
+			_("Both  Speech and Tones")
+		]

```

Comment:
@josephsl:  just making sure you saw this. :-)


On 7/1/2016 7:00 AM, James Teh wrote:
>
> In source/gui/settingsDialogs.py 
> <https://github.com/nvaccess/nvda/pull/6057#discussion_r69295674>:
>
> > -		self.lineIndentationCheckBox.SetValue(config.conf["documentFormatting"]["reportLineIndentation"])
> > -		settingsSizer.Add(self.lineIndentationCheckBox,border=10,flag=wx.BOTTOM)
> > +		sizer=wx.BoxSizer(wx.HORIZONTAL)
> > +		# Translators: This is the label for a combobox in the
> > +		# Document  Formatting  dialog (possible choices are Off, Speech, Tones, or Both.
> > +		sizer.Add(wx.StaticText(self,wx.ID_ANY,label=_("Report Line &Indents With:")))
> > +		indentChoices=[
> > +			#Translators: A setting to report No  line Indentation.
> > +			_("Off"),
> > +			#Translators: A setting to report indents with Speech.
> > +			_("Speech"),
> > +			#Translators: A setting to report indents with tones.
> > +			_("Tones"),
> > +			#Translators: A setting to report indents with both tones and Speech.
> > +			_("Both  Speech and Tones")
> > +		]
>
> @josephsl <https://github.com/josephsl>: Do you think we're going to 
> need a gettext context here? Are there conflicting messages where this 
> could cause ambiguity?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6057/files/6d6ee04d8149a704f5d806a0712cbad3fdd04cdc#r69295674>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivQ27avFRKx6Coo8sBVZRzMG4AGNPks5qRQ9wgaJpZM4IxqMD>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194